core/src/test/scala/com/broadinstitute/dsp/DbTestHelper.scala

```diff
@@ -46,7 +46,7 @@ object DBTestHelper {
     xa: HikariTransactor[IO]
   ): IO[Long] =
     sql"""INSERT INTO KUBERNETES_CLUSTER
-         (googleProject, clusterName, location, status, creator, createdDate, destroyedDate, dateAccessed, loadBalancerIp, networkName, subNetworkName, subNetworkIpRange, region, apiServerIp, ingressChart)
+         (cloudContext, clusterName, location, status, creator, createdDate, destroyedDate, dateAccessed, loadBalancerIp, networkName, subNetworkName, subNetworkIpRange, region, apiServerIp, ingressChart)
```

Comment:
same as here...

typically u wanna start PR from a clean up-to-date branch to avoid this type of issues